test/conformance/ingressv2/util.go

```diff
@@ -313,7 +313,7 @@ func CreateTimeoutService(ctx context.Context, t *testing.T, clients *test.Clien
 
 // CreateWebsocketService creates a Kubernetes service that will upgrade the connection
 // to use websockets and echo back the received messages with the provided suffix.
-func CreateWebsocketService(ctx context.Context, t *testing.T, clients *test.Clients, suffix string) (string, int, context.CancelFunc) {
+func CreateWebsocketService(ctx context.Context, t *testing.T, clients *test.Clients, suffix string) (string, gatewayv1alpha1.PortNumber, context.CancelFunc) {
```

Comment:
Do we have to change this or could we also do the conversion on the client side?